src/main/java/com/changer/marspictures/MarsPicturesController.java

```diff
@@ -0,0 +1,29 @@
+package com.changer.marspictures;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/mars/pictures")
+@RequiredArgsConstructor
+public class MarsPicturesController {
+
+    private final MarsPicturesService marsPicturesService;
+
+    @GetMapping(value = "/largest", produces = MediaType.IMAGE_JPEG_VALUE)
+    public ResponseEntity<byte[]> findLargestPicture(@RequestParam("sol") int sol,
```

Comment:
It is not mandatory to specify the name of parameter in value variable if it has the same name as method argument

Reply:
Yeah, thanks!